### PR TITLE
TW-250: Fix dismiss keyboard on scroll in new grp view

### LINKED ISSUE #250: [Chat] User cannot hide the keyboard in searching result screen of Create Group Chat flow
OS:  Android
Device: Mobile
TMail version:  2.1.3

*Precondition:  
- User create new group chat

### *Reproduce Step:
1, User create new group chat
2, User input qatest in to search box
3, User scroll up and down

### **Actual:** 
1, The cursor is not disapeared
2, The keyboard is not hide when user scroll up and down or click outside the range of keyboards

### **Expected:** 
1, The cursor will be disapeared when user scroll up and down or click on screen
2, The keyboard is hide during this action - it will be displayed with cursor only when user tap on searchbox 

https://user-images.githubusercontent.com/124866146/249575547-885cf903-9ba9-40b5-8bef-62915f0df4bf.mp4

### PR DESCRIPTION
**Pull request by imGok**
demo_scroll

This Closes #250

### CODE REVIEW

**Review by dab246 — APPROVED** [review]
lgtm

File: lib/pages/dialer/pip/dismiss_keyboard.dart [hunk 1]
```diff
@@ -1,5 +1,9 @@
 import 'package:flutter/material.dart';
 
 void dismissKeyboard(BuildContext context) {
-  FocusScope.of(context).requestFocus(FocusNode());
+  final FocusScopeNode currentFocus = FocusScope.of(context);
```
Comment (drminh2807) [inline lib/pages/dialer/pip/dismiss_keyboard.dart:4]:
```suggestion
  FocusManager.instance.primaryFocus?.unfocus();
```
No need context anymore

**Review by drminh2807 — CHANGES_REQUESTED** [review]

File: lib/pages/dialer/pip/dismiss_keyboard.dart (hunk 1 above)
Comment (imGok) [reply lib/pages/dialer/pip/dismiss_keyboard.dart]: thank u, I modified it

File: lib/pages/new_group/selected_contacts_map_change_notiifer.dart [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
   final Map<PresentationContact, ValueNotifier<bool>> selectedContactsMap = {};
   final haveSelectedContactsNotifier = ValueNotifier(false);
 
-  Iterable<PresentationContact> get contactsList 
+  Iterable<PresentationContact> get contactsList
     => selectedContactsMap.keys
       .where((contact) => selectedContactsMap[contact]?.value ?? false);
 
-  void onContactTileTap({required PresentationContact contact}) {
+  void onContactTileTap({required PresentationContact contact, required BuildContext context}) {
```
Comment (hoangdat) [inline lib/pages/new_group/selected_contacts_map_change_notiifer.dart:14]: all are `required`, why need `{ }`? I will take time of the dev :))). a small point.

**Review by drminh2807 — APPROVED** [review]

**Review by hoangdat — APPROVED** [review]

**Review by hoangdat — APPROVED** [review]

**Pull request merged**